src/components/Header.tsx

```diff
@@ -0,0 +1,133 @@
+import { useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Types';
+import { showError } from '../helpers/helpers';
+import { updateTodo, addTodos, USER_ID } from '../api/todos';
+
+interface Props {
+  todos: Todo[],
+  setError: React.Dispatch<React.SetStateAction<string>>;
+  setTempTodo: (tempTodo: Todo | null) => void;
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  setLoader: React.Dispatch<React.SetStateAction<number[]>>;
+}
+
+export const Header: React.FC<Props> = ({
+  todos,
+  setError,
+  setTempTodo,
+  setTodos,
+  setLoader,
+}) => {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [submitDisable, setSubmitDisable] = useState(false);
+  const activeInputButton = todos.every(todo => todo.completed);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    (setSearchQuery(event.target.value));
+  };
+
+  const newTodo = {
+    id: 0,
+    title: searchQuery,
+    userId: 10881,
```

Comment:
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/96138503/e9bc192a-94ca-4a7d-93ea-a6814af1023d)
you already have a variable with this id, let's reuse it here🥲